R/safe2x2Test.R

```diff
@@ -781,19 +746,16 @@ simulateFisherSpreadSampleSizeOptionalStopping <- function(deltaDesign, alpha, p
           na1 <- sum(stats::rbinom(na, 1, mu.a))
           nb1 <- sum(stats::rbinom(nb, 1, mu.b))
 
-          # TODO(Rosanne): Dit heb ik uit elkaar getrokken. Klopt dit nog?
           somePValue <- stats::fisher.test(x = matrix(c(na - na1, na1, nb - nb1, nb1),
                                                       byrow = TRUE, nrow = 2))$p.value
-
           reject.F[m] <- somePValue <= alpha
         }
 
         percent.reject.F[i] <- mean(reject.F)
       }
 
       if (min(percent.reject.F) >= power) {
-        # TODO(Rosanne): Hier heb ik FALSE van gemaakt klopt dat?
-        not.satisfied <- F
+        not.satisfied <- FALSE
```

Comment:
The dot notation is also not recommended anymore. I use a dot only to mimic `t.test` and `cor.test`, but those are aliases.